cpp/include/utilities/cython.hpp

```diff
@@ -162,26 +151,40 @@ struct graph_container_t {
 //
 // FIXME: Should local_* values be void* as well?
 void populate_graph_container(graph_container_t& graph_container,
-                              legacyGraphTypeEnum legacyType,
                               raft::handle_t const& handle,
-                              void* offsets,
-                              void* indices,
+                              void* src_vertices,
+                              void* dst_vertices,
                               void* weights,
-                              numberTypeEnum offsetType,
-                              numberTypeEnum indexType,
+                              void* vertex_partition_offsets,
+                              numberTypeEnum vertexType,
+                              numberTypeEnum edgeType,
                               numberTypeEnum weightType,
                               int num_vertices,
                               int num_edges,
```

Comment:
This code will break with more than 2B (2^31-1) vertices and/or edges.

Reply:
done, will be passing as `size_t` in latest commit.